### PR TITLE
Finish cleaning up setup*

### PR DESCRIPTION
**Pull request by BradleySappington**
Remove setup* and put flake8 in tox.ini

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -5,6 +5,8 @@ envlist =
     py39-legacy-test
     py{39,310,311}-{poppydev,pysiafdev}-cov
 
+isolated_build = true
```
Comment (zacharyburnett) [inline tox.ini:8]:
```suggestion
```
this is no longer read by Tox 4: https://tox.wiki/en/latest/upgrading.html#isolated-environment-on-by-default

File: tox.ini [hunk 2]
```diff
@@ -58,3 +60,7 @@ deps =
     pycodestyle
 commands =
     pycodestyle webbpsf
+
+[testenv:flake8]
+exclude= .git,__pycache__, test*
+max-line-length = 120
```
Comment (zacharyburnett) [inline tox.ini:66]: I would recommend moving this to a `.flake8` file (INI format), as putting it here might be a "back of the fridge" scenario (forgotten about)

**Review by zacharyburnett — APPROVED** [review]
LGTM, minor trivial suggestions

File: tox.ini (hunk 1 above)
Comment (BradleySappington) [reply tox.ini:8]: I was getting errors after removing setup.py and this was the only thing that fixed it

File: tox.ini (hunk 2 above)
Comment (BradleySappington) [reply tox.ini:66]: I may leave here just because I suspect a "ruff" replacement will be happening eventually

**Comment by BradleySappington** [comment]
@ojustino These changes mirror that of Poppy, a review would be appreciated here as well

**Review by ojustino — APPROVED** [review]

**Pull request merged**